specification/imds/data-plane/readme.md

```diff
@@ -639,6 +648,22 @@ directive:
       - $.definitions.IdentityTokenResponse.properties.object_id
       - $.definitions.IdentityTokenResponse.properties.msi_res_id
       - $.definitions.IdentityErrorResponse.properties.error_description
+
+  - suppress: DefinitionsPropertiesNamesCamelCase
+    reason: The following properties follow the Oath2 spec, which does not use camelCase.
```

Comment:
Is it important that they do? How is the `IdentityTokenResponse` used and, especially with typed SDK lanugages like Java and C#, how would callers assign an object from this response to something of a different type? Seems an adapter is needed either way.

Reply:
I'm not sure if I understand your question. If the user is using Powershell or an http client in any language, they can just take the token string from the response and use it to authorize into a resource as a bearer token and client credential. Or they can just use MSAL or some other wrapper like you said.